packages/salesforce-adapter/src/metadata_deploy.ts

```diff
@@ -221,7 +225,8 @@ export const deployMetadata = async (
 
   log.debug('deploy result: %s', JSON.stringify(deployRes, undefined, 2))
 
-  const { errors, successfulFullNames } = processDeployResponse(deployRes)
+  const { errors, successfulFullNames } = processDeployResponse(deployRes,
+    pkg.getDeletionsPackageName())
```

Comment:
line break style nit
```suggestion
  const { errors, successfulFullNames } = processDeployResponse(
    deployRes, pkg.getDeletionsPackageName()
  )
```